source/usr/lib/mcvirt/node/node.py

```diff
@@ -79,3 +79,15 @@ def is_volume_group_set(self):
     def get_version(self):
         """Returns the version of the running daemon"""
         return VERSION
+
+    @Pyro4.expose()
+    def deadlock_escape(self):
+        """Force clear a lock to escape deadlock"""
+        self._get_registered_object('auth').assert_permission(PERMISSIONS.SUPERUSER)
+        lock = MethodLock.get_lock()
+        if lock.locked():
+            lock.release()
+            Pyro4.current_context.has_lock = False
+            return True
+        else:
```

Comment:
The ``else:`` isn't necessary, since the 'if' will return, so the ``return False`` can just be in the parent scope